bbb-common-web/src/main/java/org/bigbluebutton/presentation/handlers/AbstractCommandHandler.java

```diff
@@ -51,23 +51,28 @@ public void onStart(NuProcess nuProcess) {
 
   @Override
   public void onStdout(ByteBuffer buffer, boolean closed) {
-    if (buffer != null) {
+    if (buffer != null && buffer.remaining() > 0) {
       CharBuffer charBuffer = StandardCharsets.UTF_8.decode(buffer);
-      stdoutBuilder.append(charBuffer);
+      String chunk = charBuffer.toString();
+      stdoutBuilder.append(chunk);
+      log.debug("[{}] {}", getIdTag(), chunk.trim());
     }
   }
 
   @Override
   public void onStderr(ByteBuffer buffer, boolean closed) {
     if (buffer != null) {
       CharBuffer charBuffer = StandardCharsets.UTF_8.decode(buffer);
-      stderrBuilder.append(charBuffer);
+      String chunk = charBuffer.toString();
+      stderrBuilder.append(chunk);
+      log.error("[{}] {}", getIdTag(), chunk.trim());
     }
   }
 
   @Override
   public void onExit(int statusCode) {
     exitCode = statusCode;
+    log.info("[{}] Process exited with code {}", getIdTag(), statusCode);
```

Comment:
I think it should be `debug`, or maybe log as info only when it's not zero? what do you think?

```suggestion
    log.debug("[{}] Process exited with code {}", getIdTag(), statusCode);
```